packages/salesforcedx-vscode-core/src/commands/util/parameterGatherers.ts

```diff
@@ -1,13 +1,10 @@
-/*
```

Comment:
Why did we remove the copyright header ?

Reply:
heh whoops, good question 🙃